tests/integration/optlyplugins/lookup_error_ups.go

```diff
@@ -0,0 +1,29 @@
+/****************************************************************************
```

Comment:
Since we have so many of these let's just have a `user_profile_service` package in here